src/jit/lir.h

```diff
@@ -262,6 +262,10 @@ class LIR final
         void InsertBefore(GenTree* insertionPoint, GenTree* node);
         void InsertAfter(GenTree* insertionPoint, GenTree* node);
 
+        void InsertBefore(GenTree* insertionPoint, GenTree* node1, GenTree* node2);
```

Comment:
Basically you get something like:
```C++
GenTree*[] temp { node1, node2, node3 };
struct list { GenTree **begin, **end; };
insert(point, list { &temp[0], &temp[3] });
```
So the costs are some additional memory accesses. In the overload version some of node pointers are passed in registers on x64.